content/chronograf/v1.8/administration/import-export-dashboards.md

```diff
@@ -7,46 +7,48 @@ menu:
     parent: Administration
 ---
 
-Chronograf makes it easy to both export and import dashboards.
-Dashboard exports are simple JSON files that can be shared and imported into other Chronograf instances.
-This allows you to recreate robust dashboards without having to manually configure them from the ground up.
+Chronograf makes it easy to both export and import dashboards. Recreate robust dashboards without having to manually configure them from the ground up.
 
-[Exporting a dashboard](#exporting-a-dashboard)  
-[Importing a dashboard](#importing-a-dashboard)  
+[Export a dashboard](#export-a-dashboard)  
+[Import a dashboard](#import-a-dashboard)  
 [Required user roles](#required-user-roles)  
 
-## Exporting a dashboard
-1. Go to your "Dashboards" landing page.
-2. Hover over the dashboard you would like to export and click the "Export"
-   button that appears to the right.
+## Required user roles
+
+All users can export a dashboard. To import a dashboard, a user must have an Admin or Editor role.
+
+| Task vs Role     | Admin | Editor | Viewer |
+|------------------|:-----:|:------:|:------:|
+| Export Dashboard | ✅    | ✅     | ✅     |
+| Import Dashboard | ✅    | ✅     | ❌     |
+
+## Export a dashboard
+
+1. On the Dashboards page, hover over the dashboard you want to export, and then click the **Export**
+   button on the right.
+
+   <img src="/img/chronograf/1-6-dashboard-export.png" alt="Exporting a Chronograf dashboard" style="width:100%;max-width:912px"/>
 
-<img src="/img/chronograf/1-6-dashboard-export.png" alt="Exporting a Chronograf dashboard" style="width:100%;max-width:912px"/>
+   This downloads a JSON file containing dashboard information including template variables, cells and cell information such as the query, cell-sizing, color scheme, visualization type, etc.
 
-This downloads a JSON file containing dashboard information including template variables,
-cells and cell information such as the query, cell-sizing, color scheme, visualization type, etc.
+   > No time series data is exported with a dashboard.
+   > Exports include only dashboard-related information as mentioned above.
 
-> No time series data is exported with a dashboard.
-> Exports include only dashboard-related information as mentioned above.
+2. Rename the `.json` extention to `.dashboard`, and save the dashboard file in the `/usr/share/chronograf/canned` directory.
```

Comment:
@desa is Step 2 only required when using the command line to export dashboards (does UI take care of this behind the scenes)?

@desa, I tested [this change](https://github.com/influxdata/chronograf/pull/2593) we discussed and it doesn't seem to work in the UI. Is there an intentional reason why the dashboard file extension and required path would be different between the UI and CLI?

Want to verify that I'm not missing something before proceeding with further doc changes. Thank you!

 @desa 
Renaming it to `.dashboard` should work. Is it not working as you would expect?

 @kelseiv
@desa, that is correct. Not working as described. I renamed `test.json` to `test.dashboard`, imported the file, and the dashboard file gets renamed to `test.dashboard.json`.